data/required_tasks.yml

```diff
@@ -118,28 +124,28 @@ pipeline-required-tasks:
         - prefetch-dependencies
         - s2i-nodejs
         - sast-snyk-check
-        - sbom-json-check
```

Comment:
This is the one task I see getting removed that doesn't seem to be reflected anywhere else with a similar check, is that intentional?

Reply:
That's just the diff being weird. See how in main, that task is not in the latest entry: https://github.com/release-engineering/rhtap-ec-policy/blob/31b6ea21a3d80ef52133c0b224fda09e802e6afb/data/required_tasks.yml#L17-L29 

It is on the entry prior to it. Since I removed that older entry, then the diff got "confused".